app/src/main/AndroidManifest.xml

```diff
@@ -34,20 +34,19 @@
             android:theme="@style/Theme.Duckee">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
-
-            <!-- firebase dynamic links -->
             <intent-filter>
                 <action android:name="android.intent.action.VIEW" />
-
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-
-                <data
-                    android:host="duckee.page.link"
-                    android:scheme="https" />
+                <data android:scheme="duckee" />
+            </intent-filter>
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:host="duckee.airbridge.io" android:scheme="https" />
             </intent-filter>
```

Comment:
Added intent filters for both Airbridge scheme deeplinks (duckee://) and app links (https://duckee.airbridge.io). The android:autoVerify="true" attribute is crucial for App Links verification on Android 6.0 and above. Make sure to verify the domain ownership through Digital Asset Links.